### ISSUE TITLE
downsample and upsample gives error

### ISSUE THREAD

**Issue by dgursoy** [tomopy/tomopy#172]
Need to be checked, since we encountered an error for this after #162.

**Comment by ravescovi** [comment]
I just opened another issue, dont know if they are the same.

**Comment by dgursoy** [comment]
Ok, let's cont. at #188. This one was empty anyways.
